source/exceptions.tex

```diff
@@ -1028,6 +1028,19 @@
 \tcode{std::atexit} or \tcode{std::at_quick_exit}
 exits via an exception\iref{support.start.term}, or
 
+\item%
+when evaluation of a predicate\iref{dcl.attr.contract}
```

Comment:
"a predicate" -> "the predicate of a contract"

Reply:
Fixed.